### ISSUE TITLE
msbuild error on test-azure_msft-win10-x64-5 with node-test-binary-windows on v6.x

### ISSUE THREAD

**Issue by richardlau** [nodejs/build#1716]
`node-test-binary-windows` `vcbt2015,win10,1` builds on `test-azure_msft-win10-x64-5` for v6.x are failing to compile addons,
e.g. https://ci.nodejs.org/job/node-test-binary-windows/COMPILED_BY=vcbt2015,RUNNER=win10,RUN_SUBSET=1/24026/console

```console
05:24:12 MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "4.0".
05:24:12 gyp ERR! build error 
05:24:12 gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
05:24:12 gyp ERR! stack     at ChildProcess.onExit (c:\workspace\node-test-binary-windows\deps\npm\node_modules\node-gyp\lib\build.js:276:23)
05:24:12 gyp ERR! stack     at emitTwo (events.js:106:13)
05:24:12 gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
05:24:12 gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:219:12)
05:24:12 gyp ERR! System Windows_NT 10.0.10240
05:24:12 gyp ERR! command "c:\\workspace\\node-test-binary-windows\\Release\\node.exe" "c:\\workspace\\node-test-binary-windows\\deps\\npm\\node_modules\\node-gyp\\bin\\node-gyp" "rebuild" "--directory=test\\addons\\01_function_arguments" "--nodedir=c:\\workspace\\node-test-binary-windows"
```

AFAICT this has been happening at least as far back as we have history (12 days): https://ci.nodejs.org/computer/test-azure_msft-win10-x64-5/builds

Ping @refack because
![image](https://user-images.githubusercontent.com/5445507/53975326-788db580-40fc-11e9-8f6f-6d3e118fee99.png)

cc FYI @BethGriggs

**Comment by refack** [comment]
Hopefully fix. Testing - https://ci.nodejs.org/job/node-test-commit-windows-fanned/25281/

**Comment by richardlau** [comment]
> 
> 
> Hopefully fix. Testing - https://ci.nodejs.org/job/node-test-commit-windows-fanned/25281/

Well it's slightly different (additional available tools version) but hasn't fixed it 😞:

https://ci.nodejs.org/job/node-test-binary-windows/24284/COMPILED_BY=vcbt2015,RUNNER=win10,RUN_SUBSET=1/console
```console
16:39:29 MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "14.0", "4.0".
16:39:30 gyp ERR! build error 
16:39:30 gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
16:39:30 gyp ERR! stack     at ChildProcess.onExit (c:\workspace\node-test-binary-windows\deps\npm\node_modules\node-gyp\lib\build.js:276:23)
16:39:30 gyp ERR! stack     at emitTwo (events.js:106:13)
16:39:30 gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
16:39:30 gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:219:12)
16:39:30 gyp ERR! System Windows_NT 10.0.10240
16:39:30 gyp ERR! command "c:\\workspace\\node-test-binary-windows\\Release\\node.exe" "c:\\workspace\\node-test-binary-windows\\deps\\npm\\node_modules\\node-gyp\\bin\\node-gyp" "rebuild" "--directory=test\\addons\\01_function_arguments" "--nodedir=c:\\workspace\\node-test-binary-windows"
```

**Comment by refack** [comment]
Ok, now it's fixed.
https://ci.nodejs.org/job/node-test-binary-windows/COMPILED_BY=vcbt2015,RUNNER=win10,RUN_SUBSET=1/24288/console
